stripe-patrons-data/build.sbt

```diff
@@ -42,20 +44,9 @@ assembly / assemblyMergeStrategy := {
     oldStrategy(y)
 }
 
-lazy val deployToCode =
```

Comment:
@tomrf1 why did you have to take this out? was it fundamentally broken, or you just didn't see the value in fixing it up?

Reply:
The conclusion was that it's not compatible with snapstart, which involves creating versions of lambdas.
I found this old chat message:
<img width="630" alt="Screenshot 2024-04-10 at 15 04 35" src="https://github.com/guardian/support-frontend/assets/1513454/eb77dd2c-1c3b-48c8-b09e-da363317b5ba">

Reply:
ah got it so we would have to change this to reflect the new way riffraff works - first create and upload the assembly jar, and then either run the cloudformation update, or whatever the `aws` commands are to create a version and then use it from the state machine